SentinelOne/deep_visibility/_meta/smart-descriptions.json

```diff
@@ -134,6 +249,41 @@
                 "source": "process.executable",
                 "target": "registry.path",
                 "type": "modified registry key"
+            },
+            {
+                "source": "user.name",
+                "target": "host.name",
+                "type": "logged on"
+            },
+            {
+                "source": "user.name",
+                "target": "host.name",
+                "type": "logged on"
+            },
```

Comment:
I can't spot the difference between these, are they duplicate ? 👀

Reply:
Nice catch !